src/pages/static/MarkdownReader.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { string } from 'prop-types';
+import remark from 'remark';
+import remark2react from 'remark-react';
+
+class MarkdownReader extends React.Component {
```

Comment:
Maybe we should move this component to `components/MarkdownReader.js`, we might want to use it somewhere else.

Reply:
I've moved it to `components/markdownReader/MarkdownReader.js`, in line with other reusable components.

Reply:
@dlofran2 I hope this hasn't messed things up for you

Reply:
Thanks!